presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentServer.java

```diff
@@ -53,7 +53,7 @@ public static void main(String[] args)
             injector.getInstance(Announcer.class).start();
         }
         catch (Exception e) {
-            LOG.error(e, "Error running application");
+            LOG.error(e, "Error starting server");
```

Comment:
This should be in the last commit of the PR ("changes from code review"), but I can't rewrite the history any more besides that last commit.